src/components/ClearRouteButton/index.jsx

```diff
@@ -0,0 +1,8 @@
+function ClearRouteButton() {
+    return (
+        <button className='font-Anton italic text-2xl border-2 border-red-700 rounded-3xl p-2 active:bg-amber-50'>Clear
+            Route</button>
```

Comment:
Any reason this is on a new line?

Reply:
php storm code clean up put it on a new line so it didn't go over the character limit